### ISSUE TITLE
Error: Unexpected token on Toast static propTypes

### ISSUE THREAD

**Issue by EAfang** [galio-org/galio#190]
Getting error on Toast static propTypes

`
/node_modules/galio-framework/src/Toast.js 11:19
Module parse failed: Unexpected token (11:19)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 
| class Toast extends Component {
>   static propTypes = {
|     children: PropTypes.node.isRequired,
|     isShow: PropTypes.bool.isRequired,
`

**Comment by tilldomke** [comment]
Same Problem here. Can anyone help?

**Comment by AndyOsei** [comment]
Kindly provide a reproducible example so I could look into it...

**Comment by tilldomke** [comment]
The Error occurs when I try to run the app in any way via `expo start` after `npm install galio-framework`. It happens with a new, clean app and with my app already in development

**Comment by AndyOsei** [comment]
I don't think this is an issue with Galio, it because of your babel loader config and/or webpack if you are using it. Try clearing your bundler cache and make sure your babel.config.js or web pack.config.js if any is using the appropriate config rules.

**Comment by syky27** [comment]
I am facing the same issue when trying to build for web using expo

**Comment by RichardTheHouse** [comment]
Same, fixed this by updating app.json and including the following in the expo section.
`
    "web": {
      "build": {
        "babel": {
          "include": ["galio-framework"]
        }
      }
    }
`

**Comment by syky27** [comment]
@RichardTheHouse Thanks, I figured that out, unfortunately I am getting error with almost all of my dependencies...

**Comment by BoGnY** [comment]
same problem after installing galio-framework on existing app

**Comment by ismailfarooq** [comment]
Any Updates on this issue?
